packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsModal.tsx

```diff
@@ -131,13 +133,19 @@ const usePackSizeController = (
     )
   );
 
-  const options = packSizes.map(packSize => ({
-    label: String(packSize),
-    value: packSize,
-  }));
+  const anySize = [];
+  if (packSizes.length > 1) {
+    anySize.push({ label: t('label.any'), value: -1 });
+  }
 
-  const defaultPackSize = options[0] ?? { label: '', value: '' };
+  const options = anySize.concat(
+    packSizes.map(packSize => ({
+      label: String(packSize),
+      value: packSize,
+    }))
+  );
 
+  const defaultPackSize = options[0] ?? { label: '', value: '' };
```

Comment:
I'm wondering if we should default to a pack size that is more accurate when opening hte modal in edit mode when there are only a single pack size of lines allocated. For example 

lines A,B,C
A - Pack size: 1, allocated: 10
B - pack size 1, allocated 2
C - Pack size 10, allocated 0.

Currently the pack size will be set to `any` which is not bad, just wanted to question it

Reply:
good idea!